### PR TITLE
lock_resolver: handle pessimistic locks in BatchResolveLocks

### PR DESCRIPTION
**Pull request by MyonKeminta**
Ref: https://github.com/pingcap/tidb/issues/43243

### CODE REVIEW

File: txnkv/txnlock/lock_resolver.go [hunk 1]
```diff
@@ -238,11 +238,27 @@ func (lr *LockResolver) BatchResolveLocks(bo *retry.Backoffer, locks []*Lock, lo
 	txnInfos := make(map[uint64]uint64)
 	startTime := time.Now()
 	for _, l := range expiredLocks {
+		logutil.Logger(bo.GetCtx()).Info("BatchResolveLocks handling lock", zap.Stringer("lock", l))
```
Comment (cfzjywxk) [inline txnkv/txnlock/lock_resolver.go:241]: Should we use `logutil.Debug(...)`?

**Review by cfzjywxk — APPROVED** [review]

File: txnkv/txnlock/lock_resolver.go (hunk 1 above)
Comment (MyonKeminta) [reply txnkv/txnlock/lock_resolver.go:241]: Ah sorry. I added this when debugging and forgot to remove it. Keep it as a debug-level log might be a good idea.

**Comment by MyonKeminta** [comment]
Oops something went wrong about the tests... I'll figure it out when I have time..

**Comment by MyonKeminta** [comment]
fixed in #794

**Pull request closed without merging**